python/apps/taiga/src/taiga/permissions/services.py

```diff
@@ -204,3 +204,15 @@ async def is_view_story_permission_deleted(old_permissions: list[str], new_permi
     if "view_story" in old_permissions and "view_story" not in new_permissions:
         return True
     return False
+
+
+async def is_a_self_request(user: AnyUser, obj: Any) -> bool:
+    if user.is_anonymous:
+        return False
+
+    if isinstance(obj, User):
+        return obj == user
```

Comment:
It's nice to left it open to check two users are the same (apart from the required obj.user)